tests/scripts/installandtest.sh

```diff
@@ -3,20 +3,25 @@
 set -x
 env | sort
 mkdir -p ~/.kube
+cp /tmp/kubeconfig ~/.kube/config
 cp /var/run/secrets/ci.openshift.io/multi-stage/kubeconfig ~/.kube/config
```

Comment:
This line will generate an error message when ran locally.  Probably not the end of the world, but maybe add a comment or something that it will fail outside of the openshift ci system.  I don't think we want to actually go as far suppressing the error here in case there actually is a problem in the ci system with the kubeconfig.

Reply:
I "solved" it by doing the copy from tmp and only if it fails, do the cp from /var/run...hopefully that's better